src/test/java/catan/controllers/ctf/GameUserValidator.java

```diff
@@ -47,9 +47,9 @@ public Scenario doesntHaveAvailableAction(String action) {
         return scenario;
     }
 
-    public Scenario resourcesChanged(int brick, int wood, int sheep, int wheat, int stone) {
+    public Scenario resourcesQuantityChanged(int brick, int wood, int sheep, int wheat, int stone) {
```

Comment:
is this better:
resourcesQuantityChangedOn(int brickDiff, int woodDiff, int sheepDiff, int wheatDiff, int stoneDiff)